runtime/src/main/java/com/datastax/oss/quarkus/runtime/internal/driver/QuarkusNettyOptions.java

```diff
@@ -38,11 +38,10 @@
 import java.time.Duration;
 import java.util.concurrent.ThreadFactory;
 import java.util.concurrent.TimeUnit;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.jboss.logging.Logger;
 
 public class QuarkusNettyOptions implements NettyOptions {
-  private static final Logger LOG = LoggerFactory.getLogger(QuarkusNettyOptions.class);
+  private static final Logger LOG = Logger.getLogger(QuarkusNettyOptions.class.getName());
```

Comment:
```suggestion
  private static final Logger LOG = Logger.getLogger(QuarkusNettyOptions.class);
```

Reply:
Why?
the `getLogger` is a proxy method that does what we need:
```
public static Logger getLogger(Class<?> clazz) {
        return getLogger(clazz.getName());
}
```

Reply:
Because it's less code?